src/test/ui/missing-allocator.stderr

```diff
@@ -0,0 +1,4 @@
+error: no global memory allocator found but one is required; link to std or add #[global_allocator] to a static item that implements the GlobalAlloc trait.
+
+error: aborting due to previous error
```

Comment:
_Nitpick_
I'm not really happy with this output. I would prefer something closer to the following, but I'm not sure we can at that stage:

```
error: no global memory allocator found but one is required
note: link to `std` or add `#[global_allocator]` to a static item that implements the `GlobalAlloc` trait
note: for more information visit <BOOK 2.0 LINK>

error: aborting due to previous error
```

Reply:
How do I output a note from this code?

As to including an URL, the closest we have at the moment is https://doc.rust-lang.org/nightly/std/alloc/index.html#the-global_allocator-attribute but as you can see it includes `/nightly`, since this change has not reached stable yet.

Reply:
I have too look at this, but honestly I would not block on the perfect output precisely because it's nightly. Let's just remember to revise this before stabilizing.